static/css/kiali_style.css

```diff
@@ -14,6 +14,7 @@
 
 a {
   color: #0094DE;
+  font-size: 16px;
 }
 
```

Comment:
About this I'm worried about this apply to all the links. What do you think about to create a new class for them with a hover effect for this link ?
```html
<a href="https://groups.google.com/forum/#!forum/kiali-users" class="email-group">
  <i class="icon-envelope2"></i>
  Kiali Users group
</a>
<span class="middot">&middot;</span>
<a href="https://groups.google.com/forum/#!forum/kiali-dev" class="email-group">
  <i class="icon-envelope2"></i> Kiali Developers group
</a>
```
```css
.email-group {
  font-size: 16px;
  opacity: 0.8;
}

.email-group:hover {
  opacity: 1;
}
```
This is the demo

![a hover](https://user-images.githubusercontent.com/3019213/50275228-2b743e80-043f-11e9-8ba2-0958d213a206.gif)